src/main/java/com/myfjc/text_file_reader/Application.java

```diff
@@ -0,0 +1,165 @@
+        }
+    }
+
+    private static String[] getListOfWordsFromFile(File myLyrics) throws FileNotFoundException {
+        Scanner sc = new Scanner(myLyrics);
+        String lyrics = "";
+        while (sc.hasNextLine()) {
+            String data = sc.nextLine();
+            lyrics = lyrics.concat(data + " ");
+
+        }
+        lyrics = lyrics.replaceAll(",", "");
+        lyrics = lyrics.replaceAll("\\.", "");
+        lyrics = lyrics.replaceAll("\\?", "");
+        lyrics = lyrics.replaceAll("!", "");
+        lyrics = lyrics.toLowerCase();
+
+        return lyrics.split(" ");
+    }
+
+    private static boolean itsABadWord(String word) {
+        for (String badWord : getListOfBadWords()) {
+            if (word.equalsIgnoreCase(badWord)) {
+                return true;
+            }
+        }
+        return false;
+    }
+
+    private static String[] getListOfBadWords() {
+        return new String[]{"motherfuckin'", "biatch", "niggas", "bitch", "fuckin'", "motherfuckers",
+                "nigga", "fuck"};
+    }
+
+    private static int getIndexOfACounter(String word, List<Counter> counters) {
+        for (int i = 0; i < counters.size(); i++) {
+
+            if (word.equalsIgnoreCase(counters.get(i).getValue())) {
+                return i;
+            }
+        }
+        return -1;
+    }
+
+    private static void addNewWordOrIncreaseCount(String word, List<Counter> counters) {
+        int indexOfACounter = getIndexOfACounter(word, counters);
+        if (indexOfACounter != -1) {
+            Counter counter = counters.get(indexOfACounter);
+            counter.setCount(counter.getCount() + 1);
+
+            sortElement(indexOfACounter, counters);
+        } else {
+            Counter counter = new Counter();
+            counter.setValue(word);
+            counter.setCount(1);
+            counters.add(counter);
+        }
+    }
+
+    private static void sortElement(int indexOfACounter, List<Counter> counters) {
+        boolean sortNeeded = true;
+        while (sortNeeded && indexOfACounter > 0) {
+            Counter currentCounter = counters.get(indexOfACounter);
+            Counter previousCounter = counters.get(indexOfACounter - 1);
+            int currentCount = currentCounter.getCount();
+            int previousCount = previousCounter.getCount();
+            if (currentCount > previousCount) {
+                String tempValue = previousCounter.getValue();
+                previousCounter.setValue(currentCounter.getValue());
+                currentCounter.setValue(tempValue);
+                previousCounter.setCount(currentCount);
+                currentCounter.setCount(previousCount);
+            } else {
+                sortNeeded = false;
+            }
+            indexOfACounter--;
+        }
+        /*
+        boolean b = (indexOfACounter > 0) %% (counters.get(indexOfACounter).getCount() > counters.get(indexOfACounter - 1).getCount());
+        while (b) {
+
+        }*/
+    }
```

Comment:
no-no :) no comments in the production code

Reply:
deleted comments